### PR TITLE
python solution with unittests

### PR DESCRIPTION
**Pull request by drkennetz**
python solution with unittests

### CODE REVIEW

File: challenges/2021-04-06-intervening-verbiage/solutions/drkennetz.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    def test_count_words_between(self):
+        count = 1
+        for start, stop, res in zip(self.start_words, self.stop_words, self.res):
+            actual_result = count_words_between(start, stop, self.words)
+            if actual_result == res:
+                status = "SUCCESS"
+            else:
+                status = "FAIL"
+            print("Test %i: "%count, "Start Word: %s" %start, "Stop Word: %s" %stop, "\n", "Input Sentence: %s" %self.words, "\n",
```
Comment (parhaml) [inline challenges/2021-04-06-intervening-verbiage/solutions/drkennetz.py:67]: Stretch goal with the nicely formatted test output 💯

**Review by ericwburden — DISMISSED** [review]
Looks like it passes all the given tests!

File: challenges/2021-04-06-intervening-verbiage/solutions/drkennetz.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- JDK 11+
+"""
+
+import unittest
+
+def count_words_between(start, stop, words):
+    """ Count the number of words between the start and stop word """
+    try:
+        return words.lower().split(start.lower(), 1)[1].split(stop.lower(), 1)[0].count(" ") - 1
```
Comment (parhaml) [inline challenges/2021-04-06-intervening-verbiage/solutions/drkennetz.py:48]: split is fancy. nice.

**Review by parhaml — APPROVED** [review]

**Comment by drkennetz** [comment]
This was a quick solution that doesn't deal with partial matches. Think of:

```
'There was an igloo that I liked very much'
```
If the split word was `'I'` this would fail, I think. I'll refactor and submit. I knew this was too cheap to handle everything :D

**Comment by parhaml** [comment]
> This was a quick solution that doesn't deal with partial matches. Think of:
> 
> ```
> 'There was an igloo that I liked very much'
> ```
> 
> If the split word was `'I'` this would fail, I think. I'll refactor and submit. I knew this was too cheap to handle everything :D

I wonder if `split` accepts a regex and could include spacewordspace to ensure only whole word matching

**Comment by drkennetz** [comment]
@parhaml I actually think you can. Maybe I'll refactor to that approach so it's something different. Good call

**Review by parhaml — APPROVED** [review]

**Review by ericwburden — APPROVED** [review]

**Pull request merged**